ansible/roles/stack-sunbird/tasks/assessment-service.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Remove assessment-service
+  shell: "docker service rm assessment-service"
+  ignore_errors: yes
+  
+- name: Save configurations into an env file
+  become: yes
+  template: src=assessment-service_application.conf dest=/home/deployer/config/assessment-service.conf mode=0644
+
+- name: Remove old assessment-service.conf docker config
+  become: yes
+  shell: "docker config rm assessment-service.conf"
+  ignore_errors: yes
+  
+- name: Save as docker config
+  become: yes
+  shell: "docker config create assessment-service.conf /home/deployer/config/assessment-service.conf"
+
+- name: Deploy assessment-service
+  shell: "docker service create --with-registry-auth --replicas {{ learner_replicas }} -p 9003:9000  --name assessment-service --hostname assessment-service --reserve-memory {{ learner_reservation_memory }} --limit-memory {{ learner_limit_memory }} --limit-cpu {{ learner_limit_cpu }} --reserve-cpu {{ learner_reservation_cpu }} --health-cmd 'wget -qO- assessment-service:9000/health || exit 1' --health-timeout 3s --health-retries 3  --network application_default --config source=assessment-service.conf,target=/home/sunbird/assessment-service-1.0-SNAPSHOT/config/application.conf,mode=0644  {{hub_org}}/{{image_name}}:{{image_tag}}"
```

Comment:
Please don't use learner variables. Create new for this service and add to defaults

Reply:
changes done